toolkit/tools/imagecustomizerapi/module.go

```diff
@@ -5,37 +5,54 @@ package imagecustomizerapi
 
 import (
 	"fmt"
+	"strings"
 )
 
-type Module struct {
-	Name string `yaml:"Name"`
-}
-
-func (m *Module) IsValid() error {
-	if m.Name == "" {
-		return fmt.Errorf("name of module may not be empty")
-	}
-
-	return nil
-}
+type ModuleOptions map[string]string
 
 type Modules struct {
```

Comment:
Please add these changes to `toolkit/tools/imagecustomizer/docs/configuration.md`.